OEF-Social_Service/OEF-Social_Service.DataAccess/Data/Services/FollowService.cs

```diff
@@ -3,36 +3,164 @@
 using Neo4j.Driver;
 using OEF_Social_Service.Composition;
 using OEF_Social_Service.DataAccess.Data.Services.Interfaces;
-using System;
-using System.Collections.Generic;
-using System.Linq;
+using OEF_Social_Service.Models;
+using System.Runtime.CompilerServices;
 using System.Text;
-using System.Threading.Tasks;
+using System.Text.Json;
 
 namespace OEF_Social_Service.DataAccess.Data.Services
 {
     public class FollowService : IFollowService
     {
         private IAsyncSession _session;
-
-        private ILogger<FollowService> _logger;
-
         private string _database;
-        private readonly IDriver _driver;
+        public Person _person;
+        private IAsyncSession replicaSession;
 
         public FollowService(IDriver driver, ILogger<FollowService> logger, IOptions<ApplicationSettings> appSettingsOptions)
         {
-            _logger = logger;
             _database = appSettingsOptions.Value.Neo4jDatabase;
             _session = driver.AsyncSession(o => o.WithDatabase(_database));
+            replicaSession = driver.AsyncSession(o => o.WithDatabase(_database));
+        }
+
+        public async Task CreateUser(Person person)
+        {
+            var statementText = new StringBuilder();
+            statementText.Append("CREATE (n:Person {Id : $id, Firstname: $firstname, Lastname: $lastname, Username: $username, Email: $emailAddress, Password: $password, EnrollmentDate: $enrollmentDate, Role: $role, Institution: $institution, Theme: $theme, ResidencePlace: $residencePlace})");
+            var statementParameters = new Dictionary<string, object>
+            {
+                {"id", person.Id.ToString()},
+                {"firstname", person.FirstName},
+                {"lastname", person.LastName},
+                {"username", person.Username},
+                {"emailAddress", person.EmailAddress},
+                {"password", person.Password},
+                {"enrollmentDate", person.EnrollmentDate.ToShortDateString()},
+                {"role", person.Role},
+                {"institution", person.Institution},
+                {"theme", person.Theme},
+                {"residencePlace", person.ResidencePlace}
+            };
+
+            using (_session)
+            {
+                var query = await _session.RunAsync(statementText.ToString(), statementParameters);
+                Console.Write(query);
+                
+            }
+        }
+
+        public async Task SendRequest(string person1, string person2)
+        {
+            if(person1 == person2)
+            {
+                return;
+            }
+            else
```

Comment:
Else is niet nodig, probeer zoveel mogelijk if statement checks te doen om af te breken zonder else. Dit maakt het leesbaarder. Als voorbeeld;

```
if(person1 == person2) {
   return;
}

var ...

if (thing == false) {
   return exception();
}
```